app/views/content/become-a-further-education-teacher.md

```diff
@@ -19,18 +19,16 @@ keywords:
   - apprencticeship
```

Comment:
this is spelt wrong (I appreciate this is already in prod and not part of this PR change)!

and actually I would probably be tempted to remove it as apprenticeships in the GIT world could relate to routes into teaching?